src/proposals/20240306/SparkEthereum_20240306.t.sol

```diff
@@ -0,0 +1,447 @@
+
+        assertEq(wbtcConfigBefore.ltv,                  70_00);
+        assertEq(wbtcConfigBefore.liquidationThreshold, 75_00);
+        assertEq(wbtcConfigBefore.liquidationBonus,     107_00);
+
+        /*******************/
+        /*** WETH Before ***/
+        /*******************/
+
+        ReserveConfig memory wethConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'WETH');
+
+        assertEq(wethConfigBefore.ltv,                  80_00);
+        assertEq(wethConfigBefore.liquidationThreshold, 82_50);
+        assertEq(wethConfigBefore.liquidationBonus,     105_00);
+
+        IDefaultInterestRateStrategy wethIRSBefore = IDefaultInterestRateStrategy(
+            wethConfigBefore.interestRateStrategy
+        );
+
+        _validateInterestRateStrategy(
+            address(wethIRSBefore),
+            address(wethIRSBefore),
+            InterestStrategyValues({
+                addressesProvider:             address(poolAddressesProvider),
+                optimalUsageRatio:             wethIRSBefore.OPTIMAL_USAGE_RATIO(),
+                optimalStableToTotalDebtRatio: wethIRSBefore.OPTIMAL_STABLE_TO_TOTAL_DEBT_RATIO(),
+                baseStableBorrowRate:          0.032e27,
+                stableRateSlope1:              wethIRSBefore.getStableRateSlope1(),
+                stableRateSlope2:              wethIRSBefore.getStableRateSlope2(),
+                baseVariableBorrowRate:        0,
+                variableRateSlope1:            0.032e27,
+                variableRateSlope2:            1.200e27
+            })
+        );
+
+        /*********************/
+        /*** wstETH Before ***/
+        /*********************/
+
+        ReserveConfig memory wstethConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'wstETH');
+
+        assertEq(wstethConfigBefore.ltv,                  68_50);
+        assertEq(wstethConfigBefore.liquidationThreshold, 79_50);
+        assertEq(wstethConfigBefore.liquidationBonus,     107_00);
+
+        /***********************/
+        /*** Execute Payload ***/
+        /***********************/
+
+        GovHelpers.executePayload(vm, payload, executor);
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', pool);
+
+        /*******************/
+        /*** rETH After ****/
+        /*******************/
+
+        ReserveConfig memory rethConfigAfter = rethConfigBefore;
+
+        rethConfigAfter.interestRateStrategy = _findReserveConfigBySymbol(allConfigsAfter, 'rETH').interestRateStrategy;
+
+        rethConfigAfter.ltv                  = 79_00;
+        rethConfigAfter.liquidationThreshold = 80_00;
+
+        _validateReserveConfig(rethConfigAfter, allConfigsAfter);
+
+        IDefaultInterestRateStrategy rethIRSAfter = IDefaultInterestRateStrategy(
+            rethConfigAfter.interestRateStrategy
+        );
+
+        _validateInterestRateStrategy(
+            address(rethIRSAfter),
+            address(rethIRSAfter),
```

Comment:
Not a big deal but these can be `rethConfigAfter.interestRateStrategy` or `rethIRSAfter` can be declared as an address